### PR TITLE
Easybeefstew

### PR DESCRIPTION
**Pull request by dfryer1193**
Nomnomnom

### CODE REVIEW

File: recipes/Beef/EasyBeefStew.recipe [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - (1 tsp)   Cracked pepper
+  - (1 Tbsp)  Flour
+steps:
+  - Preheat oven to 300F
+  - Sear the beef steaks in a hot pan with some oil until a crust develops. Set
+      aside on a plate to cool.
+  - Add onion, potatoes, carrots, beer, chichen broth, worcestershire sauce,
+      peas, salt, and pepper to a dutch oven.
+  - Bring to a simmer
+  - Cut beef into 1.5 inch cubes, toss with flour.
```
Comment (szabowexler) [inline recipes/Beef/EasyBeefStew.recipe]: Huh - out of curiosity, what's the effect of searing, then dredging, then braising? I've seen some that call for dredging before the searing (e.g. http://www.reluctantgourmet.com/flouring-meat/), but I don't really feel like I know what the big difference is. Perhaps the science of cooking or the food lab has the answer...?

**Review by szabowexler — CHANGES_REQUESTED** [review]
Could you also add the updated cookbook.pdf, just so we always have all recipes present there, so we can share the link out? :)

**Comment by szabowexler** [comment]
Also, you might be interested in http://cooking.nytimes.com/recipes/1016029-wine-braised-oxtail -- I made it last night, and it was pretty good. We're going to try adapting the recipe for beef chuck instead of oxtail (the oxtail was a bit too fatty), but overall it was pretty amazing.

**Comment by dfryer1193** [comment]
I'll update the PDF when I get home!

The effect of partially searing is to let the beef keep a touch more moisture during the braising process. It's the same technique found in [this Food Lab article](http://www.seriouseats.com/2016/01/food-lab-follow-the-rules-for-the-best-all-american-beef-stew.html). Curiously, I was doing this before reading that article.

The dredging is effectively optional, you'll just end up with a slightly thinner stew than if you added no flour. It helps to prevent clumping in the flour, compared to simply adding the flour on its own.

**Comment by dfryer1193** [comment]
Updated pdf!

**Comment by szabowexler** [comment]
oof -- looks like you're going to have to merge in master, and then re-add the PDF. State changes mean you're in a bad state here.

**Comment by dfryer1193** [comment]
*sigh* let's try this again.

**Review by szabowexler — APPROVED** [review]

**Pull request merged**